examples/appshell-react-host/package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "bootstrap": "npx lerna bootstrap",
     "build": "dotenv -- lerna run --scope=react-appshell-host-components build && dotenv -- lerna run --ignore=react-appshell-host-components build",
-    "clean": "lerna run --parallel clean && rm -rf coverage appshell_registry",
+    "clean": "lerna run --parallel clean && rm -rf coverage appshell_registry ../global-runtime-manifest",
```

Comment:
Is there a distinction between the appshell_registry location and global-runtime-manifest? All roads should lead back to the registry.